test/e2e/completion_test.go

```diff
@@ -11,8 +11,7 @@ import (
 )
 
 func TestCompletion(t *testing.T) {
-	Setup(t)
-	defer Teardown(t)
+	defer Setup(t)()
```

Comment:
@whynowy @sarabala1979 observe the double `()()` syntax for funning a function that contains a defer - looks weird but is valid